iphone/TitaniumKit/TitaniumKit/Sources/Modules/TiUIWindowProxy.m

```diff
@@ -271,17 +271,24 @@ - (void)viewWillAppear:(BOOL)animated; // Called when the view is about to made
 #if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
   if ([TiUtils isIOSVersionOrGreater:@"13.0"]) {
     TiColor *newColor = [TiUtils colorValue:[self valueForKey:@"barColor"]];
+    if (newColor == nil) {
+      //Get from TabGroup
+      newColor = [TiUtils colorValue:[[self tabGroup] valueForKey:@"barColor"]];
```

Comment:
do you have to worry about [self tabGroup] being nil?

Reply:
No. In Objective C it is fine to pass message on nil object. It'll return nil.